.gitignore

```diff
@@ -53,3 +53,8 @@ coverage.xml
 # Sphinx documentation
 docs/_build/
 
+# for PyCharm.
```

Comment:
Check https://github.com/SatelliteQE/robottelo/pull/3353#issuecomment-198489015 for why these changes should be dropped.